app/src/form/FormContainer.js

```diff
@@ -11,7 +11,29 @@ import { FormState, UIState } from '../store';
 // TODO:: Allow other not supported yet
 // TODO:: Repeat group not supported yet
 
-const FormContainer = ({ forms, initialValues = {}, onSubmit }) => {
+const checkValuesBeforeCallback = (values) =>
+  Object.keys(values)
+    .map((key) => {
+      let value = values[key];
+      // check empty
+      if (
```

Comment:
@wayangalihpratama I think we can simplify this.

```bash
λ node
Welcome to Node.js v18.7.0.
Type ".help" for more information.
> const values = [false, undefined, null, 0, NaN,'']
undefined
> values.map(v => !v)
[ true, true, true, true, true, true ]
```

But from what I see here, you want to keep 0 as valid value, so:

```jsx
if (value === 0) {
   return {[key]: value}
}
if (!value) {
   return false
}
```

Reply:
Hi mas @dedenbangkit , I pushed an update for this